packages/components/tests/dummy/app/templates/components/alert.hbs

```diff
@@ -0,0 +1,844 @@
+    <strong>close</strong>
+    button (optional).</p>
+  <p class="dummy-paragraph">
+    🚨
+    <em><strong>Important</strong>: we provide only the visual styling to this alert, so other features like placement,
+      animations/transitions, and what happens on dismiss, will need to be implemented in your app (eg. with an Ember
+      addon such as
+      <a href="https://github.com/poteto/ember-cli-flash">ember-cli-flash</a>).
+    </em>
+  </p>
+</section>
+
+<section>
+  <h3 class="dummy-h3">Component API</h3>
+  <p class="dummy-paragraph" id="component-api-alert">Here is the API for the component:</p>
+
+  <dl class="dummy-component-props" aria-labelledby="component-api-alert">
+    <dt>type <code>enum</code></dt>
+    <dd>
+      <p>
+        Sets the type of alert.
+      </p>
+
+      <p>Acceptable values:</p>
+      <ol>
+        <li>page</li>
+        <li>inline</li>
+        <li>compact</li>
+        <li>toast</li>
+      </ol>
+    </dd>
+
+    <dt>color <code>enum</code></dt>
+    <dd>
+      <p>
+        Sets the color scheme for
+        <code class="dummy-code">background</code>,
+        <code class="dummy-code">border</code>,
+        <code class="dummy-code">title</code>, and
+        <code class="dummy-code">description</code>, which
+        <strong>cannot</strong>
+        be overriden.
+        <code class="dummy-code">color</code>
+        results in a default
+        <code class="dummy-code">icon</code>, which
+        <strong>can</strong>
+        be overriden.
+      </p>
+      <p>Acceptable values:</p>
+      <ol>
+        <li class="default">neutral</li>
+        <li>highlight</li>
+        <li>success</li>
+        <li>warning</li>
+        <li>critical</li>
+      </ol>
+    </dd>
+    <dt>icon <code>string | false</code></dt>
+    <dd>
+      <p>Override the default
+        <code class="dummy-code">icon</code>
+        name, which is determined by the
+        <code class="dummy-code">color</code>
+        argument.</p>
+      <p>Acceptable values: any
+        <a href="https://flight-hashicorp.vercel.app/" target="_blank" rel="noopener noreferrer">
+          Flight</a>
+        icon name or pass
+        <code class="dummy-code">false</code>
+        for no icon.</p>
+    </dd>
+
+    <dt>title <code>string</code></dt>
+    <dd>
+      <p>
+        The title text in the alert.
+        <code class="dummy-code">title</code>
+        <strong>and/or</strong>
+        <code class="dummy-code">description</code>
+        is required.
+      </p>
+    </dd>
+
+    <dt>description <code>string</code></dt>
+    <dd>
+      <p>The description text in the alert.
+        <code class="dummy-code">description</code>
+        <strong>and/or</strong>
+        <code class="dummy-code">title</code>
+        is required.
+      </p>
+    </dd>
+
+    <dt>onClose <code>function</code></dt>
+    <p>
+      The alert can be dismissed by the user. Pass in a function to display this optional parameter.
+    </p>
```

Comment:
[issue] This needs to be inside of a `<dd>` element: 
```suggestion
    <dd><p>
      The alert can be dismissed by the user. Pass in a function to display this optional parameter.
    </p></dd>
```
<img width="1230" alt="image" src="https://user-images.githubusercontent.com/4587451/166322422-2bde2c61-849e-4c7e-9825-a4083fe85a24.png">

Reply:
fixed